examples/demo-rollup/rollup_config.toml

```
@@ -4,7 +4,7 @@ start_height = 671431
 
 [da]
 # The JWT used to authenticate with the celestia light client. Instructions for generating this token can be found in the README
-celestia_rpc_auth_token = "MY.SECRET.TOKEN"
+celestia_rpc_auth_token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJwdWJsaWMiLCJyZWFkIiwid3JpdGUiLCJhZG1pbiJdfQ._gj_vbeV9QYWhZZpaHHyhbuUwucBiM0ZQK1S6eNpdjI"
```

Looks like this got committed by mistake
```suggestion
celestia_rpc_auth_token = "MY.SECRET.TOKEN"
```